### PR TITLE
Add kokoro build orb to README

### LINKED ISSUE #336: add kokoro build orb to master branch readme.md
ideally for linux, windows, mac.  Similiar to[ google-cloud-intellij.](https://github.com/GoogleCloudPlatform/google-cloud-intellij)

### PR DESCRIPTION
**Pull request by TadCordle**
Fixes #336

### CODE REVIEW

**Comment by coollog** [comment]
I'm not sure if we want to put this on our front page, since it makes it harder for people to immediately find the useful information and the build status is only really useful for ourselves. It looks like there's already a check or x next to commits to master.

**Comment by patflynn** [comment]
@coollog this is a common practice across a lot of our libraries and tools as well as across github. If you want to minimize the vertical space you could always aggregate the results into one badge that you include with your other first-line badges with a link through to the test matrix.

**Comment by coollog** [comment]
Is there a way to aggregate the badges into one badge image? (without having to set up a separate CI system like Travis)

**Comment by TadCordle** [comment]
Thoughts on this layout?

**Comment by loosebazooka** [comment]
It more concise, but it is a lot of a whitespace. Lets see if we can get anything from the kokoro team for building custom badges, so can just line them up.

**Comment by loosebazooka** [comment]
Somone picked up the bug on the kokoro team, so maybe it'll happen that we can customize the badges directly.

**Comment by TadCordle** [comment]
Customized build orbs are in, should be good for review. Lemme know if we should change the layout (it's getting kind of crowded up top).

File: README.md [hunk 1]
```diff
@@ -3,7 +3,10 @@
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.google.cloud.tools/jib-maven-plugin/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.google.cloud.tools/jib-maven-plugin)
 [![Gradle Plugin Portal](https://img.shields.io/badge/gradle%20plugin-v0.9.6-blue.svg)](https://plugins.gradle.org/plugin/com.google.cloud.tools.jib)
```
Comment (briandealwis) [inline README.md]: Could maybe update this gradle version too?  (It's odd, but the maven badge remains a 0.9.6 — it appears Github is caching these images?)

Comment (TadCordle) [reply README.md]: Oh yeah, we still need to update all the 0.9.6 occurrences to 0.9.7.

Comment (coollog) [reply README.md]: Yea, I think this is the same problem: https://github.com/softwaremill/maven-badges/issues/5

Comment (coollog) [reply README.md]: We might want to consider changing to use this:

[Maven Central](https://maven-badges.herokuapp.com/maven-central/com.google.cloud.tools/jib-maven-plugin)

Comment (briandealwis) [reply README.md]: I purged the camo cache for the Maven image with https://stackoverflow.com/questions/26898052/how-to-force-image-cache-update-in-readme-rst-on-github

**Comment by TadCordle** [comment]
Here are some ideas I have for cleaning up the top:
- Move the analytics badge to the bottom of the page; I think it's good to have, but I don't think it's important enough to be one of the first things on the page.
- Remove the gitter badge since we already have a link to it in the "Get involved with the community" section, and I don't think it's that useful to have at the top of the page since people are probably going to want to read about the project before deciding to join the gitter.
- Remove the twitter badge (@coollog mentioned that no one seems to be using it), or at least replace it with a link in the "Get involved with the community section".

**Comment by TadCordle** [comment]
Doesn't look bad without the Gitter and Twitter badge.

**Comment by coollog** [comment]
I think we should still keep the gitter badge since that's where most gitter users would look for that.

File: README.md [hunk 2]
```diff
@@ -53,10 +54,13 @@ We welcome contributions! Here's how you can contribute:
 * [Browse issues](https://github.com/GoogleContainerTools/jib/issues) or [file an issue](https://github.com/GoogleContainerTools/jib/issues/new)
 * Chat with us on [gitter](https://gitter.im/google/jib)
 * Join the [jib-users mailing list](https://groups.google.com/forum/#!forum/jib-users)
+* [Tweet about us](https://twitter.com/intent/tweet?text=Google%20container%20tool%20for%20dockerizing%20Java%20applications&url=https://github.com/GoogleContainerTools/jib&via=JibContainerBuilder&hashtags=java,docker,kubernetes,maven,gradle,microservices,jib)
```
Comment (coollog) [inline README.md]: I think we can just leave this out - tweets will happen naturally for new announcements and articles.

**Pull request merged**